helm/nifikop/crds/nifi.konpyutaika.com_nificlusters.yaml

```diff
@@ -736,6 +736,15 @@ spec:
                           type: integer
                         name:
                           type: string
+                        protocol:
+                          allOf:
+                          - default: TCP
+                          - default: TCP
```

Comment:
There is an issue with this I think.

Reply:
Interesting. This is problematic, but it was auto generated. Let me modify and see if I can fix it manually.

Reply:
It might be caused by both defining a default and the enum. kubebuilder might inject this default twice.

Reply:
It is - removing the default fixes the problem. Can't find a nice way to do this without using a defaulting webhook, which is a much larger lift. Are you okay leaving the annotation off and i can add a comment clarifying that the default will be the first value in the enum?

Reply:
Okay, fixed. I had to add `// +kubebuilder:validation:Optional` to prevent that from happening.